web-common/src/features/alerts/EditAlertDialog.svelte

```diff
@@ -45,6 +45,7 @@
       name: alertSpec.title as string,
       snooze: getSnoozeValueFromAlertSpec(alertSpec),
       recipients: alertSpec?.emailRecipients?.map((r) => ({ email: r })) ?? [],
+      splitByTimeGrain: alertSpec.intervalsIsoDuration ?? "",
```

Comment:
Let's call this `interval` to mirror the API parameter or `evaluationInterval` to mirror the form label.